install_config/aggregate_logging.adoc

```diff
@@ -849,19 +849,103 @@ and the xref:#deploying-the-efk-stack[{product-title} logging installer] creates
 Make sure you specify the correct parameters in order to use persistent volumes with EFK.
 Also set the `openshift_enable_unsupported_configurations=true` parameter in the Ansible inventory file, as the logging installer blocks the installation of NFS with core infrastructure by default.
 
+If using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provisioned PVs], the {product-title} logging installer creates PVCs that use the default storage class or the one specified with the `openshift_logging_elasticsearch_pvc_storage_class_name` parameter.
+
+If using NFS storage, the {product-title} installer creates the persistent volumes, based on the `openshift_logging_storage_*` parameters 
+and the xref:#deploying-the-efk-stack[{product-title} logging installer] creates PVCs, using the openshift_logging_es_pvc_ paramters.
+Make sure you specify the correct parameters in order to use persistent volumes with EFK.
+
+If your environment requires NFS storage, there are two methods you can employ: 
+
+* xref:aggregated-logging-nfs-persistent[NFS as persistent volume] 
+
+* xref:aggregated-logging-nfs-local[NFS storage as local storage].
+
 [WARNING]
 ====
 Using NFS storage as a volume or a persistent volume, or using NAS such as
 Gluster, is not supported for Elasticsearch storage, as Lucene relies on file
 system behavior that NFS does not supply. Data corruption and other problems can
-occur. If you must use NFS storage, allocate a large file on a
-volume to serve as a storage device and mount it locally on one host.
-For example, if your NFS storage volume is mounted at *_/nfs/storage_*:
+occur. 
+====
+
```

Comment:
@anpingli Can you take a look at the following steps that you suggested in [Logging Installer forgets to create PV when using NFS](https://github.com/openshift/openshift-docs/pull/12748#discussion_r232602303).